wpeverest/everest-forms
# Feature - Introduced Credit card field for stripe

### All Submissions:






### Changes proposed in this Pull Request:


This PR removes the payment charge option field and registered the credit card field and compatibility of stripe v3.

### How to test the changes in this Pull Request:

1. Select the form and now you can't see the payment charge option field anymore.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Introduced Credit card field and Payment charge option field is deprecated.
